backend/medtagger/api/tasks/service_rest.py

```diff
@@ -59,3 +61,18 @@ class Task(Resource):
     def get(task_key: str) -> Any:
         """Return task for given key."""
         return business.get_task_for_key(task_key)
+
+
+@tasks_ns.route('/<string:task_key>/metadata')
```

Comment:
I don't like the idea to have two endpoints for the same business entity. Let's keep only one of them and make it simple.